integration/src/test/kotlin/kotlinx/benchmark/integration/WasmGcOptionsTest.kt

```diff
@@ -2,24 +2,18 @@ package kotlinx.benchmark.integration
 
 import org.junit.Test
 
+// TODO: Remove this test and the associated code that adds the "--experimental-wasm-gc" flag once
+//  the minimum supported Kotlin version no longer supports D8 < 12.3.68.
 class WasmGcOptionsTest : GradleTest() {
-    @Test
-    fun nodeJs() {
-        // The test uses Kotlin 1.9.24 as previous versions
-        // would append the --experimental-wasm-gc flag causing run failures.
-        val runner = project(
-            "wasm-gc-non-experimental/wasm-nodejs", true, kotlinVersion = "1.9.24"
```

Comment:
Since the supported Kotlin versions are not compatible with NodeJs versions earlier than 22.0.0, I am not sure if this use case is still viable: https://github.com/Kotlin/kotlinx-benchmark/pull/213#discussion_r1598490631.

If it is, the code that adds the `--experimental-wasm-gc` flag to `NodeJsExec` should be preserved.

However, this test does not seem to make much sense in any case, since the supported Kotlin versions use NodeJs 22.0.0 or newer, and setting the `nodeVersion` to an earlier version will cause the test project to fail.

Reply:
@adam-enko Could you please help me understand if the use case mentioned is still viable with Kotlin 2.0.20?

Reply:
I tried to build a 2.0.20-based project with NodeJS `21.0.0-v8-canary-*` and after a few a options added, everything continue working the same way it does with v22.

I don't know why one would do that in their projects, but it works.